shared/chat/conversation/reply-preview/index.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Styles from '../../../styles'
+
+export type Props = {|
+  onCancel: () => void,
+  text: string,
+  username: string,
+|}
+
+const ReplyPreview = (props: Props) => {
+  return (
```

Comment:
This should exit if the message we're replying to explodes / is deleted
<img width="333" alt="image" src="https://user-images.githubusercontent.com/11968340/56741915-c0ff4200-6741-11e9-8d3f-22dc3a7f3c61.png">

Reply:
I'll fix on the server